templates/entrypoints/tmux.bash

```diff
@@ -13,4 +15,5 @@ tmux set-option -g status-right ""
 # Set unique to container background color.
 tmux set -g status-bg blue
```

Comment:
These options need to move to their own config file.

If the assumption is that tmux is part of a compliant dev_contain container I'm ok with that. I'm not sure I'm cool with that assumption though...